src/components/activity/ActivityFormPanel.svelte

```diff
@@ -167,51 +193,104 @@
 
   <svelte:fragment slot="body">
     {#if $selectedActivity}
+      <div class="activity-header">
+        <div class="activity-header-title-placeholder st-typography-medium">
+          {activityName}
+        </div>
+      </div>
       <fieldset>
```

Comment:
Can we just remove this outer fieldset?

Reply:
Yeah, will need to add padding to some wrapper element.